processor/processor.go

```diff
@@ -177,6 +183,9 @@ func (proc *HandleT) addJobsToSessions(jobList []*jobsdb.JobT) {
 			}
 			proc.userPQItemMap[userID] = pqItem
 			proc.userJobPQ.Add(pqItem)
+			logger.Debug("Adding a new session to userID ", userID)
+			// Adding a new session id for the user
+			proc.userToSessionIDMap[userID] = pqItem.lastTS.Unix()
```

Comment:
There could be multiple users who have the same session ID. You might want to prepend timestamp with the userID. Another option is to generate UUID everytime.

Reply:
since it's just a identifier, it won't be an issue.
Moving to uuid would be better, will do

Reply:
Doesn't the transformer need to know that the same identifier could be used for different sessions? 
I agree it might not be a problem now. But going forward, someone who is writing a user/dest transform function might not know this can happen. It's always good to have session_id to be unique.

Reply:
For a user, across sessions, session_id is different.
Agree, it should be unique each time for different users as well. Will make it uuid